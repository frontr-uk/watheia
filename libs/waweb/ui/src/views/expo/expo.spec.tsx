import { render } from '@testing-library/react';

import ExpoView from './expo';

describe('Expo', () => {
  it('should render successfully', () => {
    const { container } = render(<ExpoView />);
    expect(container).toBeInstanceOf(HTMLElement);
    expect(container).toMatchSnapshot();
  });
});
