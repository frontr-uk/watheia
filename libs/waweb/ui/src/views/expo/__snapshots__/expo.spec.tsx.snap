// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expo should render successfully 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Welcome to Expo!
    </h1>
  </div>
</div>
`;
